Makefile

```diff
@@ -81,11 +81,18 @@ $(GENERATE_UNBOUND_METHODS_EXAMPLE_SRCS): $(GENERATE_UNBOUND_METHODS_EXAMPLE_SPE
 	@rm -f $(EXAMPLE_CLIENT_DIR)/generateunboundmethods/README.md \
 		$(EXAMPLE_CLIENT_DIR)/generateunboundmethods/git_push.sh
 
+TMP_INSTALL_DIR := $(shell mktemp -d)
 install:
+
+install:
```

Comment:
Wait, one of these should go right?

Reply:
Makefile fixed.

I've tried to check the dependencies but no obvious link to the large amount of changes, must be a dependency difference between building protoc-gen-go and protoc-gen-go-grpc with the local `go.mod` vs building it from their tree.

As a side effect I've also bumped the version of `google.golang.org/grpc/cmd/protoc-gen-go-grpc` v1.0.0 -> v1.1.0